src/sos-packaging.props

```diff
@@ -56,11 +50,25 @@
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\libsos.dylib" TargetRid="osx-arm64" />
     <SosRequiredBinaries Condition="'$(PackageWithCDac)' == 'true'" Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\libmscordaccore_universal.dylib" TargetRid="osx-arm64" />
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\sosdocsunix.txt" TargetRid="osx-arm64" />
+
+    <_SosBinariesToPack Include="@(SosRequiredBinaries)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <_winSosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\sos.dll" TargetRid="win-x64" />
+    <_winSosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\Microsoft.DiaSymReader.Native.amd64.dll" TargetRid="win-x64" />
+    <_winSosRequiredBinaries Condition="'$(PackageWithCDac)' == 'true'" Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\mscordaccore_universal.dll" TargetRid="win-x64" />
+
+    <!-- When we are running the official build and packaging, don't copy too much. We already publish sos.dll in another leg, so favor that. -->
+    <_SosBinariesToPack Condition="'$(BuildX64Package)' == 'true'" 
+                        Include="@(_winSosRequiredBinaries)" />
+
+    <SosRequiredBinaries Include="@(_winSosRequiredBinaries)" />
   </ItemGroup>
 
   <!-- What and where to pack SOS assets in the final packages. -->
   <ItemGroup>
-    <None Include="@(SosRequiredBinaries)">
+    <None Include="@(_SosBinariesToPack)">
```

Comment:
Would there be a way to condition the CopyToOutputDirectory on whether or not the file exists in the path?

How does the other copy get published, is it before this step or after? We'd need to react accordingly in case the "publish sos.dll in another leg" gets removed.

Reply:
I suspect to do that in msbuild would lead to more convoluted steps. I'd rather just conditionally never add it to the item as opposed to compare and reduce.